views/requests.erb

```diff
@@ -130,12 +136,14 @@
             <% end %>
           </td>
           <td>
+            <span aria-hidden="true">Pickup date:</span>
             <%= booking_request.booking_date %>
           </td>
           <td>
+            <span aria-hidden="true">Expires:</span>
             <%= booking_request.booking_date %>
```

Comment:
@niquerio `Line 140` and `Line 144` are pulling in the same method. Is this intentional?

Reply:
same deal as above